build/ci/evergreen.yml

```
@@ -1044,6 +1044,16 @@ tasks:
           permissions: public-read
           content_type: ${content_type|application/x-gzip}
           display_name: downloads-center-
+      - command: subprocess.exec
+        type: test
+        params:
+          working_dir: src/github.com/mongodb/mongocli
+          silent: true
+          include_expansions_in_env:
+            - evergreen_user
+            - evergreen_api_key
+            - slack_channel
```

[nit] just in case we later add another kind of notification
```suggestion
            - release_slack_channel
```